app/components/Arena/EventCard.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import React from "react";
+import { useRouter } from "next/navigation";
+
+import Button from "../Button";
+
+const EventCard = ({ title, description, image, links }: any) => {
+  const router = useRouter();
+
+  return (
+    <div
+      className="mb-6 flex w-[90vw] max-w-md cursor-pointer flex-col items-start justify-start transition-transform duration-300 hover:scale-105"
+      onClick={() => router.push(links[0].url)}
+    >
+      <img className="h-48 w-full object-cover" src={image} alt={title} />
```

Comment:
Why not use [`next/image`](https://nextjs.org/docs/app/building-your-application/optimizing/images)?